diff_drive_controller/src/diff_drive_controller.cpp

```diff
@@ -642,6 +651,19 @@ namespace diff_drive_controller
         last_odom_tf_publish_time_ + publish_period_ < time + half_period &&
         tf_odom_pub_->trylock())
     {
+      // Update twist:
```

Comment:
Actually, with the current implementation there's no problem if I simply remove this:
* The odom TF doesn't use the twist at all, only the (absolute/global) pose.
* The (absolute/global) pose is computed from the previous (absolute/global) pose, ie. it doesn't use the (internal) incremental pose in the current implementation, so it's always up to date with the latest control cycle, and can be published consistently at any publish rate (because it doesn't depend on any `dt`, like the twist).